src/fu-util.c

```diff
@@ -188,6 +194,36 @@ fu_util_client_notify_cb (GObject *object,
 			       fwupd_client_get_percentage (priv->client));
 }
 
+static void
+fu_util_perhaps_show_unreported (FuUtilPrivate *priv)
+{
+	if (priv->unreported_failed == 0 && priv->unreported_success == 0)
+		return;
+	g_printerr ("\n******************\n\n");
+	if (priv->unreported_failed > 0) {
+		g_printerr ("%s: %s\n",
+			    /* TRANSLATORS: we failed to apply a firmware update */
+			    _("WARNING"),
+			    /* TRANSLATORS: explain why we want to upload */
+			    ngettext ("A firmware update failed to be applied",
+				      "Firmware updates failed to be applied",
+				      priv->unreported_failed));
+	} else if (priv->unreported_success > 0) {
+		g_printerr ("%s: %s\n",
+			    /* TRANSLATORS: we did a firmware update well */
+			    _("INFO"),
+			    /* TRANSLATORS: explain why we want to upload */
+			    ngettext ("A firmware update was applied successfully",
+				      "Firmware updates were applied successfully",
+				      priv->unreported_success));
+	}
+	g_printerr ("%s\n > fwupdmgr history-report\n",
+		    /* TRANSLATORS: what the user has to do, command follows */
+		    _("To share useful information with the developers and "
+		      "clear this message use the following command:"));
```

Comment:
If nagging like this, probably need a way to turn off the nag as well.  How about tell them to modify `/etc/fwupd/client.conf` and change a `PromptUnsubmittedReports` there.

Reply:
I figured a better way to do this would be to disable (comment out) the ReportURI and check that is a valid URI before doing the nagging.

Reply:
That works too, probably need to mention it in the nag then.